imports/plugins/core/ui/client/components/table/sortableTable.js

```diff
@@ -0,0 +1,292 @@
+import React,  { Component } from "react";
+import PropTypes from "prop-types";
+import _ from "lodash";
+import matchSorter from "match-sorter";
+import ReactTable from "react-table";
+import { Translation } from "@reactioncommerce/reaction-ui";
+import { SortableTableFilter, SortableTablePagination } from "./sortableTableComponents";
+
+class SortableTable extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentPage: 0,
+      filterInput: "",
+      maxPages: 0,
+      query: this.props.query || {}
+    };
+
+    this.handleFilterInput = this.handleFilterInput.bind(this);
+  }
+
+
+  /**
+   * getMeteorData() - Absorb publication / collection information from props, output data from subscription
+   * @prop {String} matchingResultsCount - Send to Counts collection to get results count of sub
+   * @prop {String} publication - publication to subscribe to
+   * @prop {Object} collection - collection to get data from
+   * Use props to get collection, EmailTableColumn
+   * Use that info to call meteor and get subscription
+   * Output data for table
+   * @returns {Object} loading status (bool), results (object), and matchingResults (number)
+   */
+  getMeteorData() {
+    const { collection, matchingResultsCount, publication } = this.props;
+
+    // Get a count of the number of items matching the current filter.
+    // If no filter is set it will return the total number of items in the collection.
+    const matchingResults = Counts.get(matchingResultsCount);
+
+    const options = {};
+    let skip;
+
+    const pubHandle = Meteor.subscribe(publication, this.state.query, _.extend({
+      skip: skip
```

Comment:
We are using `skip` right after we declare it as `undefined`. Is that what we want to do?